mmv1/products/spanner/Database.yaml

```diff
@@ -120,8 +120,6 @@ properties:
       the values 1d, 24h, 1440m, and 86400s are equivalent. Default value is 1h.
       If this property is used, you must avoid adding new DDL statements to `ddl` that
       update the database's version_retention_period.
-    validation: !ruby/object:Provider::Terraform::Validation
-      function: ValidateDatabaseRetentionPeriod
```

Comment:
Is 30d already allowed in the backend? 

I wonder if the description of this field should be updated.

Reply:
30d is not something generally available for all customers but for some limited customers go/cs-30-day-pitr, but because of handwritten validation in Terraform they were not able to use it.

I think we should not update the description of this field because it holds true for most of cloud spanner customers.